CONTRIBUTING.md

```diff
@@ -1,203 +1,337 @@
+```
 
-## Adding Translations to Documentation
-Adding translations to our project's documentation helps make it more accessible to a wider audience. If you're interested in contributing translations for our documentation, follow these steps
-Steps to Add Translations
-1. Identify Target Languages:
-- Determine which languages you want to add translations for. Make sure these languages are relevant to the project's user base.
-2. Create Translation Files:
-- Inside the documentation directory, create a new subdirectory for each language you plan to support. Use language codes (e.g., "en" for English, "fr" for French) as directory names.
+#### 6. Blockquotes
+
+Create blockquotes using the greater-than symbol (`>`).
 
+```markdown
+> This is a blockquote.
+```
+
+#### 7. Code Blocks
+
+Create code blocks by using triple backticks (` ``` `) for fenced code blocks and specify a language next to the backticks before the fenced code block to highlight the syntax.
+
+````
 ```bash
-- docs/
-  - en/         (English)
-  - fr/         (French)
-  - es/         (Spanish)
+git pull
 ```
-3. Translate Content:
-- For each language directory, create translated versions of the documentation files. Typically, you will translate Markdown files, but consider other formats as needed.
-- Maintain the same file structure and filenames as in the original documentation, but with translated content.
-4. Update Links:
-- In the translated files, ensure that any internal links (e.g., links to other sections or pages within the documentation) are updated to point to the corresponding translated content.
-5. Add Language Selector:
-- Consider adding a language selector to the documentation, allowing users to switch between different language versions. You can do this by modifying the navigation or header of your documentation.
-6. Testing and Validation:
-- Test the translated documentation to ensure accuracy and readability. Make sure that all links work correctly and that the content is culturally appropriate.
-7. Submit Contributions:
-- If you haven't already, submit your translations as a pull request. Ensure that you provide clear information about the languages you've translated and any specific details related to your contributions.
-8. Collaborate and Review:
-- Collaborate with other contributors and reviewers to ensure the quality of translations. Be open to feedback and suggestions for improvement.
-
-Best Practices for Translations
+````
+
+### Markdown Tips
+
+- Preview your Markdown locally to ensure proper formatting before submitting your contribution.
+- Keep your Markdown content organized, and use headings to structure your sections.
+- Use code blocks to highlight code snippets or configuration examples.
+- Check out the official [Markdown Guide](https://www.markdownguide.org/basic-syntax/) website for more information about using Markdown.
+
+## Adding Translations to Documentation
```

Comment:
As we have [i18n-guidelines.md](https://github.com/open-sauced/intro/blob/main/i18n-guidelines.md) file, I'm thinking to move this section into that translation guidelines and link that guide in this section.

Reply:
Good idea @adiati98! :)